main.js

```diff
@@ -129,3 +132,38 @@ autoUpdater.on('update-downloaded', (ev, info) => {
 app.on('ready', function()  {
   autoUpdater.checkForUpdates();
 });
+
+function fixMeasureUnit(value, type, precision) {
```

Comment:
This is great -- thank you for this PR!  But since this is supposed to be a minimal example, could you take this function out?  In my experience with this repo, every little bit of extra code causes confusion.  I had no idea how much confusion :)